### ISSUE TITLE
Custom access file UI

### ISSUE THREAD

**Issue by gjost** [densho/ddr-local#247]
See [Custom access file import/update](https://github.com/densho/ddr-cmdln/issues/125)
See [Custom access file publication](https://github.com/densho/ddr-cmdln/issues/127)

Simply replacing the usual `File.access_rel`, created by appending `-a.jpg` to `File.path_rel`, with a custom filename doesn't change the Editor UI or the data emitted by `ddrindex publish`.

Custom access files need to be reflected in these template lines at least:
```
collections/detail.html       47:{% thumbnail collection.signature_abs
entities/detail.html          56:{% thumbnail entity.signature_abs
files/detail.html             42:{% thumbnail file.access_abs

collections/entities.html     68:{% thumbnail entity.signature_abs
entities/children.html        77:{% thumbnail entity.signature_abs
entities/files.html          100:{% thumbnail file.access_abs
files/edit.html               32:{% thumbnail file.access_abs

collections/list-object.html   4:{% thumbnail object.signature_url
entities/list-object.html      4:{% thumbnail object.signature_url
files/list-object.html         4:{% thumbnail object.access_url

search/search.html           155:{% thumbnail object.links.img
```

Custom access files must also be emitted by `ddrindex publish`

**Comment by gjost** [comment]
Note: We don't have to modify the templates because we are replacing the access file with another file of the same name.  Templates don't have to change.

**Comment by GeoffFroh** [comment]
Fixed in da0ed121a0a9a6fb81dd99a234fc0c48135d9a1d (?)
